pytest_django/asserts.py

```diff
@@ -0,0 +1,30 @@
+"""
+Dynamically load all Django assertion cases and expose them for importing.
+"""
+from django.test import (
+    TestCase, SimpleTestCase,
+    LiveServerTestCase, TransactionTestCase
+)
+
+test_case = TestCase('run')
+
+
+def _wrapper(name):
+    def assertion_func(*args, **kwargs):
+        getattr(test_case, name)(*args, **kwargs)
+
+    return assertion_func
+
+
+__all__ = []
+asserts = set()
+asserts.update(
+    set(attr for attr in vars(TestCase) if attr.startswith('assert')),
+    set(attr for attr in vars(SimpleTestCase) if attr.startswith('assert')),
+    set(attr for attr in vars(LiveServerTestCase) if attr.startswith('assert')),
+    set(attr for attr in vars(TransactionTestCase) if attr.startswith('assert')),
+)
+
+for assert_func in asserts:
+    locals()[assert_func] = _wrapper(assert_func)
```

Comment:
locals() is not meant to be always settable, maybe juste use globals() to make it clearer for newcomers?